command/forceleave/forceleave.go

```diff
@@ -1,31 +1,48 @@
-package command
+package forceleave
 
 import (
+	"flag"
 	"fmt"
-	"strings"
+
+	"github.com/hashicorp/consul/command/flags"
+	"github.com/mitchellh/cli"
 )
 
-// ForceLeaveCommand is a Command implementation that tells a running Consul
-// to force a member to enter the "left" state.
-type ForceLeaveCommand struct {
-	BaseCommand
+func New(ui cli.Ui) *cmd {
+	c := &cmd{UI: ui}
+	c.init()
+	return c
+}
+
+type cmd struct {
+	UI    cli.Ui
+	flags *flag.FlagSet
+	http  *flags.HTTPFlags
+	help  string
+}
+
+func (c *cmd) init() {
+	c.flags = flag.NewFlagSet("", flag.ContinueOnError)
+	c.http = &flags.HTTPFlags{}
+	flags.Merge(c.flags, c.http.ClientFlags())
+	flags.Merge(c.flags, c.http.ServerFlags())
+	c.help = flags.Usage(help, c.flags, c.http.ClientFlags(), c.http.ServerFlags())
```

Comment:
This is an agent-level command so should just have `ClientFlags` but not `ServerFlags`.

Reply:
done.